src/api/include/migraphx/migraphx.hpp

```diff
@@ -684,6 +684,60 @@ inline program parse_onnx_buffer(const std::string& buffer)
         own{});
 }
 
+/// Options for parsing tf options
+struct tf_options : MIGRAPHX_HANDLE_BASE(tf_options)
+{
+    tf_options() { this->make_handle(&migraphx_tf_options_create); }
+
+    tf_options(migraphx_tf_options* p, own) { this->set_handle(p, own{}); }
+
+    /// Make tf parser treat an inputs with a certain dimensions
+    void set_input_parameter_shape(const std::string& name, std::vector<std::size_t> dim)
+    {
+        call(&migraphx_tf_options_set_input_parameter_shape,
+             this->get_handle_ptr(),
+             name.c_str(),
+             dim.data(),
+             dim.size());
+    }
+
+    /// Change data layout to NHWC (default is NCHW)
+    void set_nhwc(bool is_nhwc)
```

Comment:
It should be `void set_nhwc(bool is_nhwc=true)`

Reply:
fixed